metarecord/views/classification.py

```diff
@@ -6,20 +6,32 @@
 
 from .base import HexRelatedField
 
+INCLUDE_RELATED_QUERY_PARAM = 'include_related'
+
 
 class ClassificationSerializer(serializers.ModelSerializer):
     id = serializers.UUIDField(source='uuid', format='hex', read_only=True)
     parent = HexRelatedField(read_only=True)
-    phases = serializers.SerializerMethodField(method_name='_get_phases')
 
     class Meta:
         model = Classification
         fields = ('id', 'created_at', 'modified_at', 'code', 'title', 'parent', 'description', 'description_internal',
-                  'related_classification', 'additional_information', 'function_allowed', 'phases')
+                  'related_classification', 'additional_information', 'function_allowed')
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        request = self.context['request']
+        self.include_related = request.query_params.get(INCLUDE_RELATED_QUERY_PARAM)
+
+    def get_fields(self):
+        fields = super().get_fields()
+
+        if self.include_related:
```

Comment:
Isn't this value always a string, so this will be true also for query `?include_related=0` ?

Also, I would like the filter to accept `true` and `True` as truthy values as well to be more consistent with most of HKI APIs.

Reply:
Alright. I wasn't aware of this convention in HKI APIs. I'll change it to accept `true` and `True`. 👍